docs/rfc/009-bit-byte-conversions.md

```diff
@@ -8,237 +8,330 @@ The Aleo Team.
+More precisely:
+* The conversion of a group element (_x_, _y_) to big endian bits is `[...x, ...y]`,
+  where `x` is the conversion of _x_ to big endian bits and `y` is the conversion of _y_ to big endian bits.
+* The conversion of a group element (_x_, _y_) to little endian bits is `[...y, ...x]`,
+  where `x` is the conversion of _x_ to little endian bits and `y` is the conversion of _y_ to little endian bits.
+
+Reversing the order of the coordinates for little endian means that
+the little endian conversion of a group element is the reverse of the big endian conversion of the same group element,
+a property shared with the integer and field element conversions.
+However, given that a group element is not a single number,
+the notion of big and little endian as such does not directly apply to group elements,
+but only to their individual coordinates.
+
+Not all (little or big endian) sequences of `N` bits represent group elements.
+Not only each `N/2`-bit half must represent a field element (see discussion for field elements above),
+but also the resulting point (_x_, _y_) must satisfy the curve equation.
+Attempting to convert to a group element a sequence of `N` bits that does not actually represent a group element
+causes an error, in the same sense as division by zero causes an error.
+
+Conversions with byte sequences are handled by juxtaposition of the byte sequence representations of _x_ and _y_:
+* The conversion of a group element (_x_, _y_) to big endian bytes is `[...x, ...y]`,
+  where `x` is the conversion of _x_ to big endian bytes and `y` is the conversion of _y_ to big endian bytes.
+* The conversion of a group element (_x_, _y_) to little endian bytes is `[...y, ...x]`,
+  where `x` is the conversion of _x_ to little endian bytes and `y` is the conversion of _y_ to little endian bytes.
+
+For the currently supported elliptic curve in Leo, `N` is 506, and `M` is 64.
+These valuee will change with the curve, when Leo is independently extended with support for more curves,
+as discussed earlier for field elements.
+
+### Conversions with Characters
+
+A character is a Unicode code point, i.e. an integer between 0 and 10FFFFh.
+Thus, it can be viewed as a 21-bit unsigned integers,
+with bit conversions defined in a natural way,
+and byte coversions defined by adding 3 most significant zero bits to reach 24 bits, which is 3 bytes.
+
+However, currently characters are represented as field elements under the hood,
+and this causes the conversions to use the same number of bits and bytes as the field conversions,
+namely `N` is 253 and `M` is 32.
+
+A character can be always converted to a (little or big endian) sequence of bits or bytes.
+However, analogously to field and group elements,
+attempting to convert to character a sequence whose numeric value exceeds 10FFFFh
+causes an error in Leo.
+This is the case also if `N` and `M` are reduced to be 21 and 3.
```

Comment:
This sentence is not clear.  What is being proposed exactly?  Should there be an inequality here?

Reply:
I just pushed some edits that attempt to clarify. If that seems fine, you can resolve this conversation.